forward_engineering/helpers/updateHelper.js

```diff
@@ -489,28 +489,35 @@ const getCommentedDropScript = (scriptsData, data) => {
 	if (applyDropStatements) {
 		return scriptsData;
 	}
-	return scriptsData.map((scriptData = {}) => {
-		if (!scriptData.deleted || !scriptData.script) {
-			return scriptData;
+	return scriptsData.map((dto = {}) => {
+		if (!dto?.scripts[0]?.isDropScript || !dto?.scripts[0]?.script) {
+			return dto;
 		}
 		return {
-			...scriptData,
-			script: commentDeactivatedStatement(scriptData.script, false),
+			...dto,
+			scripts: dto.scripts.map(scriptObject => {
```

Comment:
activation status is not taken into account in commenting out process

Reply:
getCommentedDropScript function does this - I added the activate/deactivate logic there